src/fa/check.py

```diff
@@ -24,7 +24,13 @@ def map_(mapname, force=False, silent=False):
     if fa.maps.isMapAvailable(mapname):
         logger.info("Map is available.")
         return True
-
+ 
+    if "neroxis_map_generator" in mapname:
```

Comment:
Please check if map is generated by matching against a regex. See java client MapGeneratorServiceImpl for that regex.